docker/images/n8n-custom/Dockerfile

```diff
@@ -6,7 +6,7 @@ FROM --platform=linux/amd64 n8nio/base:${NODE_VERSION} AS builder
 # Build the application from source
 WORKDIR /src
 COPY . /src
-RUN --mount=type=cache,id=pnpm-store,target=/root/.local/share/pnpm/store --mount=type=cache,id=pnpm-metadata,target=/root/.cache/pnpm/metadata pnpm install --frozen-lockfile
+RUN --mount=type=cache,id=pnpm-store,target=/root/.local/share/pnpm/store --mount=type=cache,id=pnpm-metadata,target=/root/.cache/pnpm/metadata CI=1 pnpm install --frozen-lockfile
```

Comment:
instead of having to add the `CI` env in multiple places, maybe we can use [this](https://github.com/n8n-io/n8n/blob/master/packages/cli/src/services/frontend.service.ts#L404) check instead ?

Reply:
Sure, I'll check that it works 👍

Reply:
Did not manage to get it working on both linux and mac. Used a custom env var instead